### ISSUE TITLE
NEXT: New create-skeleton-app and Skeleton CLI

### ISSUE THREAD

**Issue by endigo9740** [skeletonlabs/skeleton#2355]
> [!WARNING] 
> This issue is a work in progress.

**Comment by Hugos68** [comment]
What's the plan with this issue?

**Comment by endigo9740** [comment]
Keep in mind these are in future milestones, which mean tasks that are to be discussed post-release. But general idea was this was to propose the future for the CSA and the potential for adding a dedicated CLI. Now that we have the CLI, we should likely retool this to discuss future expansion of the CLI itself.

**Comment by Hugos68** [comment]
Yeah that's mainly why I asked since I wasn't sure if it was targeting what to do with the CLI now or what to do with a future CLI (which exists now)

**Comment by endigo9740** [comment]
I've renamed the ticket to make this a bit more clear.

**Comment by endigo9740** [comment]
Feature suggestion: automatic preset generation for custom colors....

https://github.com/skeletonlabs/skeleton/issues/2964#issuecomment-2707688441
